### ISSUE TITLE
plan to put in julia register 

### ISSUE THREAD

**Issue by ksaritek** [ylxdzsw/RedisAlchemy.jl#2]
We plan to use your client. do you plan to put in julia registry? also is there a reason why do you put blob.jl and bit.jl in a comment.

**Comment by ylxdzsw** [comment]
Yes, but we need to add some tests before submit a register application to METADATA.jl.
I haven't use blob.jl and bit.jl for a while and I'm not sure if they still works . They can be uncommented after adding proper test.
